addOns/reports/src/main/zapHomeFiles/reports/high-level-report/report.html

```diff
@@ -0,0 +1,331 @@
+
+<link rel="stylesheet" th:href="${resources + '/main.css'}"
+	href="resources/main.css">
+<link rel="stylesheet" th:href="${resources + '/bootstrap.min.css'}"
+	href="resources/bootstrap.min.css">
+<link rel="stylesheet" th:href="${resources + '/normalize.css'}"
+	href="resources/normalize.css">
+
+</head>
+<body>
+	<!-- Header material for report template -->
+	<div class="container">
+		<div class="row"></div>
+		<div class="row">
+			<div class="mx-auto" style="width: 50%;">
+				<th:block>
+					<h1>
+						<!-- The ZAP Logo -->
+						<img th:src="${resources + '/zap32x32.png'}"
+							src="resources/zap32x32.png" alt="">
+						<th th:text="${reportTitle}">Report Title</th>
+					</h1>
+				</th:block>
+			</div>
+		</div>
+		<div class="row"></div>
+		<div class="row">
+			<em
+				th:text="#{report.generated(${#dates.format(new java.util.Date(), 'EEE, d MMM yyyy HH:mm:ss')})}"
+				th:class="'text-right'">Date, Time</em>
+
+		</div>
+	</div>
+	<p></p>
+
+	<!-- Description, and first row of charts-->
+	<div class="container ">
+		<th:block
+			th:with="sites_size=${reportData.sites == null ? 0 : reportData.sites.size}, more_than_one_sites=${sites_size &gt; 1}">
+
+			<div class="row">
+				<div class="col piChartWrapper styled_canvas">
+					<th:block
+						th:if="${more_than_one_sites} and ${reportData.isIncludeSection('vulnerabilityCountPerSiteChart')} and ${#strings.arraySplit(description, '\n').length} &gt; 0">
+						<h4 th:text="#{report.template.reportDesc}">Report Desc</h4>
+						<ul>
+							<th:block
+								th:each="desc, state: ${#strings.arraySplit(description, '\n')}">
+								<li th:text="${desc}"></li>
+							</th:block>
+						</ul>
+					</th:block>
+				</div>
+			</div>
+			<p></p>
+
+			<div class="row">
+				<div class="col-md-2 col-lg-2 col-sm-3 col-xs-12">
+					<div class="row">
+						<p></p>
+					</div>
+					<div class="row border-bottom" th:with="highest_alert=''">
+						<b>Most severe alert level:</b> <br> <span
+							th:if="${alertCounts.get(3) == null? 0 : alertCounts.get(3)} &gt; 0"
+							th:text="'High'" th:class="'risk-3'"></span> <span
+							th:if="${alertCounts.get(2) == null? 0 : alertCounts.get(2)} &gt; 0 and ${alertCounts.get(3) == null? 0 : alertCounts.get(3)} == 0"
+							th:text="'Medium'" th:class="'risk-2'"></span> <span
+							th:if="${alertCounts.get(1) == null? 0 : alertCounts.get(1)} &gt; 0 and ${alertCounts.get(2) == null? 0 : alertCounts.get(2)} == 0"
+							th:text="'Low'" th:class="'risk-1'"></span> <span
+							th:if="${alertCounts.get(0) == null? 0 : alertCounts.get(0)} &gt; 0 and ${alertCounts.get(1) == null? 0 : alertCounts.get(1)} == 0"
+							th:text="'Informational'" th:class="'risk-0'"></span>
+					</div>
+					<div class="row" th:if="${more_than_one_sites}">
+						<p></p>
+					</div>
+					<div class="row border-bottom" th:if="${more_than_one_sites}">
+						<b>Asset at highest risk:</b> <br> <span
+							id="asset_with_most_critical_bug">XXXXX</span>
```

Comment:
Just ran the report with multiple sites and noticed this - presumably the `XXXXX` should have been replaced?

Reply:
Oh yeah! will remove that, as I do the suggested i18n. Will return the implementation once the data can be availed